test/itkFrequencyBandPassImageFilterTest.cxx

```diff
@@ -72,12 +72,10 @@ int itkFrequencyBandPassImageFilterTest( int, char*[] )
   TEST_SET_GET_VALUE( passBand, bandPassFilter->GetPassHighFrequencyThreshold() );
 
   bool passLowFreqThreshold = true;
-  bandPassFilter->SetPassLowFrequencyThreshold( passLowFreqThreshold );
-  TEST_SET_GET_VALUE( passLowFreqThreshold, bandPassFilter->GetPassLowFrequencyThreshold() );
+  TEST_SET_GET_BOOLEAN( bandPassFilter, PassLowFrequencyThreshold, passLowFreqThreshold );
```

Comment:
That macro is not defined.
I recommend to compile after any change, are you able to? The windows build is now passing in the ITK dashboard.